app/views/public/api_documentation.html.erb

```diff
@@ -544,6 +545,49 @@
 
 <hr />
 
+<h2 class="h4" id="gossip-nodes">Gossip Nodes</h2>
+<p>Gossip Nodes endpoint returns solana rpc nodes.</p>
+
+<h3 class="h6">Request:</h3>
+<pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/gossip-nodes/:network'</pre>
+
+<h3 class="h6">Parameter Options:</h3>
+<p><strong class="text-muted">`staked=(true | false)`</strong> will limit the results only to staked / unstaked nodes. </p>
+<pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/gossip-nodes?staked=true'</pre>
+<p><strong class="text-muted">`per=NN`</strong> will limit the results to NN entries per page, defaults to 100.</p>
+<pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/gossip-nodes/:network?per=50'</pre>
+<p><strong class="text-muted">`page=NN`</strong> will return the results from passed page, defaults to 1.</p>
+<pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/gossip-nodes/:network?page=1'</pre>
+
+<h3 class="h6">Response:</h3>
+<pre>
+[
+  {
+		"account": "ArGzEq1cp3L3dtDKgdCzBXZjdM3XHa9P5chvT79m2gGv",
+		"ip": "108.171.202.250",
+		"network": "mainnet",
+		"staked": true,
+		"software_version": "1.10.35",
+		"created_at": "2022-08-22T07:07:17.933Z",
+		"avatar_url": null,
+		"details": null,
+		"keybase_id": null,
+		"name": null,
+		"updated_at": "2022-08-12T09:45:29.459Z",
```

Comment:
Where is this date coming from? It's not node's updated_at date. And it's earlier than creation date which looks super weird and confusing (another example below). In this endpoint I think we should only return timestamps of main record (gossip node in this case)
```
[{
"account":"245B9WFHUGuWycSXHagHXwsXGcxDkNYfxWBaeh7vAHDU",
"ip":"65.21.94.8",
"network":"mainnet",
"staked":true,
"software_version":"1.10.35",
**"created_at":"2022-08-22T06:47:59.402Z",**
"avatar_url":null,
"details":"-",
"keybase_id":"",
"name":"aaanonymous",
**"updated_at":"2022-08-08T07:10:05.071Z",**
"www_url":"",
"admin_warning":null,
"country_name":"Finland",
"data_center_key":"24940-FI-Helsinki",
"latitude":"60.179",
"longitude":"24.934"
}]
```